MIGRATION_V1_V2.md

```diff
@@ -23,8 +23,14 @@ should be used as a checklist when converting a piece of code using PyLops from
   must now provide the directions along which the operator is applied through `axes`. The default value for `axis` and
   `axes` are chosen to be -1 and (-2, -1), respectively, whereas the default `dir` and `dirs` was 0 and (0, 1), respectively.
   Be careful to check all operators where `dir`, `dirs` and `nodir` was not provided explicitly.
-- `dims_fft` in the FFT operators is deprecated in favor of `dimsd`.
+### Basic
 - `dims_d` in `Sum` is deprecated in favor or `dimsd`
+### Signal processing
+- `dims_fft` in the FFT operators is deprecated in favor of `dimsd`.
+### Wave-equation processing
+- The optional arguments ``fast``, ``transpose``, and ``dtype`` have been deprecated in ``pylops.waveeqprocessing.mdd.MDC``.
+  As previously stated in a warning message, the recommended option ``transpose=True`` is now selected as default.
+  Ensure that the input array ``G`` is organized as follows ``[n_fmax X n_s X n_r]``.
```

Comment:
Just a question, wouldn't the fastest way be `ns, nr, nf`? The fast dimension here is frequency, no. Anyways, just a thought.